central/globaldb/metrics/metrics.go

```diff
@@ -177,6 +179,20 @@ var (
 		Name:      "postgres_table_toast_bytes",
 		Help:      "bytes being used by toast for a table",
 	}, []string{"Table"})
+
+	PostgresDBSize = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.CentralSubsystem.String(),
+		Name:      "postgres_active_db_size",
+		Help:      "bytes being used by Postgres Database",
+	})
+
+	PostgresConnected = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
It would be more helpful to have the accumulated number of flapping or the frequency of the disconnect.

Reply:
@c-du for alerting purposes, I think the 0/1 is best for that purpose.  What I would propose then is an additional metric such as `Disconnects` that is a running count.

Reply:
@c-du I took a look at this.  I think for what we need now so the ACSCS team can alert on the condition that the flag best serves that purpose.  I'll gladly looking at some info on tracking frequency as we build out the rest of ROX-14171